src/main/java/bio/terra/service/dataset/IngestRequestValidator.java

```diff
@@ -22,6 +35,36 @@ private void validateTableName(String name, Errors errors) {
     }
   }
 
+  public List<String> validateIngestParams(IngestRequestModel ingestRequestModel, UUID datasetId) {
+    Dataset dataset = datasetService.retrieve(datasetId);
+    CloudPlatformWrapper platform =
+        CloudPlatformWrapper.of(dataset.getDatasetSummary().getStorageCloudPlatform());
+
+    if (platform.isAzure()) {
+      return validateAzureIngestParams(ingestRequestModel);
+    }
+    return Collections.emptyList();
```

Comment:
```suggestion
    return List.of();
```

Reply:
Any reason for one over the other?

Reply:
`List.of()` is a more general form since it supports zero, one or many args. It's also nice in that it's in `List` instead of `Collections`, so it's more tied to the type we're using. So I've been using `List.of()` (and `Map.of()`, `Set.of()` etc) wherever I used to use the collection methods.

But that's it, it's not more efficient or better in any measurable way. So either way is really fine.

Reply:
This actually becomes a non-issue since I'm changing the way errors are handled!